libPhoneNumber-GeocodingParser/libPhoneNumber-GeocodingParser/NBGeocoderMetadataParser.h

```diff
@@ -0,0 +1,22 @@
+//
+//  NBGeocoderMetadataParser.h
+//  libPhoneNumber-GeocodingParser
+//
+//  Created by Rastaar Haghi on 7/1/20.
+//  Copyright © 2020 Google LLC. All rights reserved.
+//
+
+#import "NBSQLiteDatabaseConnection.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface NBGeocoderMetadataParser : NSObject
+
+- (instancetype)initWithDestinationPath:(NSURL *)desiredDatabaseLocation;
```

Comment:
Make the param name destinationPath to match

Reply:
done.